scripts/travis/build-all-platform-binaries.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Builds all platform binaries.
+set -e
+
+BRANCH=${BRANCH:?'missing BRANCH env var'}
```

Comment:
You can probably fallback to `git branch --show-current`.

Reply:
Done.